### DISCUSSION TITLE
Tailwind theme

### DISCUSSION THREAD

**Discussion by gerardp** [saadeghi/daisyui#3065] [Ideas / feature requests]
I really like the approach of DaisyUI, but I think the default theme of Tailwind is better. Would it be possible to have a theme like Tailwind's? With the same colors and sizes for buttons, forms, etc.?

**Comment by saadeghi** [comment]
What do you mean by `default theme of Tailwind`? Because Tailwind provides every possible color and every possible spacing value. It's totally agnostic about colors, spacing and sizes of buttons, forms, etc. It gives you all the possible values and you can use them as utility classes.

**Reply by gerardp** [reply]
Something like the examples shown in:
https://tailwindui.com/

**Reply by saadeghi** [reply]
That's not the "default Tailwind CSS theme". They are opinionated blocks from Tailwind UI (TailwindUI is from the same team but it's a different product than TailwindCSS)  

Apart from layout, spacing, font size, etc (which you still need Tailwind CSS utility classes to make them) smaller parts like buttons, form elements, etc are so similar to daisyUI. You can simply create a new daisyUI theme with your preferred colors so buttons, forms, etc would look similar to the examples at TailwindUI.
